packages/amplication-client/src/Resource/git/AppGitStatusPanel.tsx

```diff
@@ -75,10 +75,9 @@ const AppGitStatusPanel = ({ resource, showDisconnectedMessage }: Props) => {
             <div className={`${CLASS_NAME}__last-sync`}>
               <Icon icon="clock" />
               <Tooltip
-                aria-label={`Last sync: ${format(lastSync, DATE_FORMAT)}`}
+                aria-label={`Last sync: never`}
```

Comment:
1. Why did you remove the `format` method?
2. please don't use hard-coded "never". It is "never" only when there is no connection to GitHub, but if the user did sync with GitHub, the last sync should be a valid date with a specific format.

Reply:
Fine..I'm on it.